### PR TITLE
Fix various errors in sv translations

### PR DESCRIPTION
**Pull request by lokal-profil**
Fixed some typos and several entries which were translations of the
wrong English homonym.

Also added "Cash machine" as a (Brittish) English alias for ATM.

### CODE REVIEW

**Comment by lokal-profil** [comment]
Anything I can do to help get this merged?

**Comment by burivuh** [comment]
Hi! We'll definitely check this out! Sorry for such a delay: we had a very busy month

File: data/categories.txt [hunk 1]
```diff
@@ -7186,7 +7190,7 @@ pl:^Wejście
 pt:^Entrada
 ro:^Intrare
 es:^Entrada
-sv:^Entré
+sv:^Entré|ingång
```
Comment (mpimenov) [inline data/categories.txt]: The rest seem ok, but here I'd like to double check.
The symbol '^' in this file marks the translation that will be shown in the UI. Is it true that "Entré" is more commonly used than "Ingång"?

**Comment by lokal-profil** [comment]
Largely depends on the context.
In general "entré" is a slightly larger "ingång" and can also refer to the area imediately inside the door.

Ingång can be any entrance or opening.

**Pull request merged**